app/controllers/organizations_controller.rb

```diff
@@ -229,4 +229,29 @@ def get_inactive_days(membership, period_start)
                end
     days_off.to_i
   end
+
+  def get_members_inactive_days
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Lint/DuplicateMethods: Method OrganizationsController#get_members_inactive_days is defined at both app/controllers/organizations_controller.rb:208 and app/controllers/organizations_controller.rb:233.